kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -77,8 +77,24 @@ usageRightsEditor.controller(
 
     // setting our initial values
     editsApi.getUsageRightsCategories().then(cats => {
-        ctrl.categories = cats;
-        ctrl.originalCats = cats;
+        const copy = angular.copy(cats);
+
+        copy.forEach(cat => {
+            cat.properties.forEach((property, i) => {
+                let propertyOptions = property.required ? [] : [{key: 'None', value: null}];
```

Comment:
Putting `Null` option in the options because https://github.com/angular/angular.js/issues/8670.